src/scss/yellow/gaiden.scss

```diff
@@ -1,6 +1,5 @@
 @import 'node_modules/reset-css/_reset'; //sass-lint:disable-line clean-import-paths
 @import '../shared/niten-tokens';
-@import 'settings/colors';
```

Comment:
Are you sure this line should be removed? :thinking:

Reply:
Ops! 😬